docs/table_design/Data_distribution.md

```diff
@@ -1,107 +1,209 @@
+| List partitioning                     | A typical scenario is to query and manage data based on emurated values, and a partition needs to include data with different values for each partitioning column. For example, if you frequently query and manage data based on countries and cities, you can use this method and select `city` as the partitioning column. So a partition can store data for multiple cities belonging to the same country. | Created manually                           |
+
+**How to choose partitioning columns and granularity**
+
+- Selecting a proper partitioning column can effectively reduce the amount of data scanned during queries. In most business systems, partitioning based on time is commonly adopted to resolve issues caused by the deletion of expired data and facilitate management of tiered storage of hot and cold data. In this case, you can use expression partitioning or range partitioning and specify a time column as the partitioning column. Additionally, if the data is frequently queried and managed based on emurated values, you can use expression partitioning or list partitioning and specify a column including these values as the partitioning column.
+- When choosing the partitioning granularity, you need to consider factors data volume, query patterns, and data management granularity.
+  - Example 1: If the monthly data volume in a table is small, partitioning by month can reduce the amount of metadata compared to partitioning by day, thereby reducing the resource consumption of metadata management and scheduling.
+  - Example 2: If the monthly data volume in a table is large and queries mostly request data of certain days, partitioning by day can effectively reduce the amount of data scanned during queries.
+  - Example 3: If the data needs to expire on a daily basis, partitioning by day is recommended.
+
+#### Bucketing
+
+The bucketing method divides a partition into multiple buckets. Data in a bucket is referred to as a tablet.
+
+The supported bucketing methods are [random bucketing](#random-bucketing-since-v31) (from v3.1) and [hash bucketing](#hash-bucketing).
+
+- Random bucketing: When creating a table or adding partitions, you do not need to set a bucketing key. Data within a partition is randomly distributed into different buckets.
+
+- Hash Bucketing: When creating a table or adding partitions, you need to specify a bucketing key. Data within the same partition is divided into buckets based on the values of the bucketing key, and rows with the same value in the bucketing key are distributed to the corresponding and unique bucket.
+
+The number of buckets: By default, StarRocks automatically sets the number of buckets (from v2.5.7). And you can also manually set the number of buckets. For more information, please refer to [determining the number of buckets](#determine-the-number-of-buckets).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/c12c8e8d-c077-48aa-8ae4-8902b04d497e)
Unnecessary. Suggest removing it.